terraform/20-app/locals.tf

```diff
@@ -32,7 +32,19 @@ locals {
 
   use_ip_allow_list = local.environment != "prod"
 
-  scheduled_scaling_policies_for_non_essential_envs = {
+  dpd_dev_env_scheduled_policy = {
+    start_of_working_day_scale_out = {
+      min_capacity = 1
+      max_capacity = 1
+      schedule     = "cron(0 07 ? * MON-FRI *)" # Run every weekday at 7am
```

Comment:
Cron schedules are correct but this will probably run as UTC? Do we want to adjust for UK time?

Reply:
You're absolutely right -> [43c27af](https://github.com/UKHSA-Internal/data-dashboard-infra/pull/1291/commits/43c27afd752614f12b43c96a3b388b32d6eff13d)

Reply:
Perfect